eng/common/pipelines/templates/steps/credscan.yml

```diff
@@ -0,0 +1,41 @@
+parameters:
+  SuppressionFilePath: 'eng/CredScanSuppression.json'
+  BaselineFilePath: ''
+  SourceDirectory: $(Build.SourcesDirectory)
+
+steps:
+- pwsh: |
+    New-Item -Path "${{ parameters.SourceDirectory }}" -Name "credscan.tsv"
+    if ("$(Build.Reason)" -eq 'PullRequest') {
+      New-Item -Path "${{ parameters.SourceDirectory }}" -Name "changedFiles" -ItemType "directory"
```

Comment:
Why do we need a new folder for changedFiles? We are no longer copying them so I don't think that is needed.